library/xmltraduit2.sax.po

```diff
@@ -0,0 +1,289 @@
+"as *filename_or_stream*, can be a filename or a file object.  The *handler* "
+"parameter needs to be a SAX :class:`~handler.ContentHandler` instance.  If "
+"*error_handler* is given, it must be a SAX :class:`~handler.ErrorHandler` "
+"instance; if omitted,  :exc:`SAXParseException` will be raised on all "
+"errors.  There is no return value; all work must be done by the *handler* "
+"passed in."
+msgstr ""
+"Créer un analyseur SAX et l'utiliser pour analyser un document. Le document transmis"
+"comme * filename_or_stream *, peut être un nom de fichier ou un objet fichier."
+"Le paramètre *handler* doit être une instance SAX :class:`~handler.ContentHandler`. "
+"Si un * error_handler * est donné, il doit être un SAX :class:`~handler.ErrorHandler` "
+"; si omis, :exc:`SAXParseException` sera levé sur toutesles "
+"erreurs. Il n'y a pas de valeur de retour, tout le travail doit être fait par le *handler*"
+"transmis."
+
+#: ../Doc/library/xml.sax.rst:53
+msgid ""
+"Similar to :func:`parse`, but parses from a buffer *string* received as a "
+"parameter.  *string* must be a :class:`str` instance or a :term:`bytes-like "
+"object`."
+msgstr ""
+"Similaire à :func:`parse`, mais qui analyse à partir d'un buffer * string * reçu en tant que "
+"paramètre. * string * doit être une instance :class:`str` ou un objet :term:`bytes-like`."
+
+#: ../Doc/library/xml.sax.rst:57
+msgid "Added support of :class:`str` instances."
+msgstr ""
+"Ajout du support des instances :class:`str`."
+
+#: ../Doc/library/xml.sax.rst:60
+msgid ""
+"A typical SAX application uses three kinds of objects: readers, handlers and "
+"input sources.  \"Reader\" in this context is another term for parser, i.e. "
+"some piece of code that reads the bytes or characters from the input source, "
+"and produces a sequence of events. The events then get distributed to the "
+"handler objects, i.e. the reader invokes a method on the handler.  A SAX "
+"application must therefore obtain a reader object, create or open the input "
+"sources, create the handlers, and connect these objects all together.  As "
+"the final step of preparation, the reader is called to parse the input. "
+"During parsing, methods on the handler objects are called based on "
+"structural and syntactic events from the input data."
+msgstr ""
+"Une application SAX typique utilise trois types d'objets: les readers ,les handlers et"
+"les sources d'entrée.  \"Reader\"dans ce contexte est un autre terme pour l'analyseur, c'est-à-dire "
+"un morceau de code qui lit les octets ou les caractères de la source d'entrée"
+"et qui produit une séquence d'événements .Les événements sont ensuite distribués aux"
+"objets du handler, c'est-à-dire que le lecteur appelle une méthode sur le handler."
+"L'application doit donc obtenir un objet reader, créer ou ouvrir les sources d'entrée, "
+"créer les handlers et connecter ces objets tous ensemble."
+"La dernière étape de la préparation, le reader est appelé à analyser l'entrée."
```

Comment:
Je préfèrerai « reader » en italique si tu le laisse en anglais.

Reply:
C'est noté.